serving/samples/helloworld-scala/README.md

```diff
@@ -0,0 +1,107 @@
+# Hello World - Scala using Akka HTTP sample
+
+A microservice which demonstrates how to get set up and running with Knative Serving when using [Scala](https://scala-lang.org/) and [Akka](https://akka.io/) [HTTP](https://doc.akka.io/docs/akka-http/current/). It will respond to a HTTP request with a text specified as an `ENV` variable named `MESSAGE`, defaulting to `"Hello World!"`.
+
+##Prerequisites
+
+* A Kubernetes cluster [installation](https://github.com/knative/docs/blob/master/install/README.md) with Knative Serving up and running.
+* [Docker](https://www.docker.com) installed locally, and running, optionally a Docker Hub account configured or some other Docker Repository installed locally.
+* [Java JDK8 or later](https://adoptopenjdk.net/installation.html) installed locally.
+* [Scala's](https://scala-lang.org/) standard build tool [sbt](https://www.scala-sbt.org/) installed locally.
+
+## Configuring the sbt build
+
+Have a look at the [build.sbt](build.sbt) and **specify your Docker repository name**:
+
+```
+// If you want to use your Docker Hub repository, set this to "docker.io/yourusername/yourreponame".
+// If you use Minikube and want to use the Docker Repository inside Minikube, either set this to "dev.local" or
```

Comment:
I'd put a newline (paragraph break) between the Docker Hub directions and the minikube instructions, and then wrap these preformatted paragraphs at 80 chars? (The preview scrolls off the screen here: https://github.com/knative/docs/blob/f8e323da51d983189c20567c4f8df9cdfd74aebc/serving/samples/helloworld-scala/README.md#configuring-the-sbt-build)

Reply:
That makes a lot of sense. I'll fix!